app/services/e_invoices/ubl/base_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module EInvoices
+  module Ubl
+    class BaseService < EInvoices::BaseService
```

Comment:
I think this entire folder to be extracted from the `services` folder. 

I think we could consider this to be a serializer so maybe `serializers/xml/ubl`.
It could also be `models/ubl` or `support/ubl`.

But I think it's should definitely be outside `services`. It doesn't return a result (and I'm sure it shouldn't!).